benchmarks/build.gradle

```diff
@@ -13,15 +13,18 @@ configurations {
 dependencies {
   // Use the baseline to avoid using new APIs in the benchmarks
   compileOnly libs.reactor.perfBaseline.core
+  compileOnly libs.reactor.perfBaseline.coreMicrometer
```

Comment:
I'm not too familiar with Gradle, so happy to receive feedback if I should have modified this file (or `gradle/libs.versions.toml` below) in some other way.

Reply:
Yep, this line together with the versions catalog are ok. I'll comment below what needs to be adjusted.